src/typing/constraint_js.ml

```diff
@@ -2167,6 +2167,40 @@ let reasons_of_trace ?(level=0) ?(tab=2) trace =
     else []
   in f level trace
 
+
```

Comment:
Nop, indeed it's not, I messed up, I have to take time to fix it because I fear to break something :/